### PR TITLE
UN-587: Focused article page

### PR DESCRIPTION
**Pull request by jamesbiggs**
Ticket URL: https://national-archives.atlassian.net/browse/UN-587

## About these changes

Created a focused article page type, which is essentially a stripped down version of the article page type

## How to check these changes

Create a focused article page and make sure it includes all the fields listed in the ticket.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: etna/core/models/mixins.py [hunk 1]
```diff
@@ -133,4 +138,38 @@ class Meta:
...
+    )
+
+    hero_image_caption = RichTextField(
+        verbose_name="hero image caption (optional)",
+        features=["bold", "italic", "link"],
+        blank=True,
+        help_text=(
+            "An optional caption for hero images. This could be used for image sources or for other useful metadata."
+        ),
+    )
```
Comment (ababic) [inline etna/core/models/mixins.py:160]:
```suggestion
class RequiredHeroImageMixin(HeroImageMixin):
    """Mixin to add hero_image attribute to a Page, and make it required."""
```

File: etna/core/models/mixins.py [hunk 2]
```diff
@@ -133,4 +138,38 @@ class Meta:
...
+    content_panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel("hero_image"),
+                FieldPanel("hero_image_caption"),
+            ],
+            heading="Hero image",
+        )
+    ]
+
```
Comment (ababic) [inline etna/core/models/mixins.py:174]: Inherited from HeroImageMixin, so...

```suggestion
    content_panels = [
        MultiFieldPanel(
            [
                FieldPanel("hero_image"),
                FieldPanel("hero_image_caption"),
            ],
            heading="Hero image",
        )
    ]
```

File: templates/articles/focused_article_page.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% extends 'base_page.html' %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+{% load sections_tags %}
+{% load wagtailmetadata_tags %}
```
Comment (ababic) [inline templates/articles/focused_article_page.html:6]:
```suggestion
{% load static wagtailcore_tags sections_tags wagtailmetadata_tags %}
```

File: templates/articles/focused_article_page.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+{% block meta_tag %}
+  {% meta_tags %}
+{% endblock %}
+{% block breadcrumb %}{% endblock %}
+{% block content %}
```
Comment (ababic) [inline templates/articles/focused_article_page.html:11]:
```suggestion

{% block meta_tag %}
  {% meta_tags %}
{% endblock %}

{% block breadcrumb %}{% endblock %}

{% block content %}
```

File: etna/core/models/mixins.py (hunk 2 above)
Comment (jamesbiggs) [reply etna/core/models/mixins.py:174]: Hi Andy, I'm not sure if you've tried these on your local - but when I tried to do this it didn't work for some reason - I kept getting a load of errors. I'll give them another go now though but I did think it was odd that I couldn't just inherit it this way..

Comment (jamesbiggs) [reply etna/core/models/mixins.py:174]:
```python
SystemCheckError: System check identified some issues:

ERRORS:
articles.ArticlePage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
articles.ArticlePage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TimePeriodExplorerIndexPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent 
model 'wagtailcore.page'.
collections.TimePeriodExplorerIndexPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TimePeriodExplorerPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
collections.TimePeriodExplorerPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TopicExplorerIndexPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
collections.TopicExplorerIndexPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager
collections.TopicExplorerPage: (models.E005) The field 'id' from parent model 'core.requiredheroimagemixin' clashes with the field 'id' from parent model 'wagtailcore.page'.
collections.TopicExplorerPage: (wagtailcore.E002) Manager does not inherit from PageManager
        HINT: Ensure that custom Page managers inherit from wagtail.models.PageManager

**Review by ababic — CHANGES_REQUESTED** [review]
Looking good, @jamesbiggs! Just a few tidy-up bits here to look at

File: etna/core/models/mixins.py (hunk 1 above)
Comment (jamesbiggs) [reply etna/core/models/mixins.py:160]: Hi @ababic, same comment as this one: https://github.com/nationalarchives/ds-wagtail/pull/970#discussion_r1221444578

**Review by ababic — APPROVED** [review]
LGTM!

**Pull request merged**